heartbeat/Squid.in

```diff
@@ -114,6 +116,14 @@ for a squid instance managed by this RA.
 <content type="string" default="${OCF_RESKEY_squid_conf_default}"/>
 </parameter>
 
+<parameter name="squid_opts" required="0" unique="1">
```

Comment:
You should change it to `unique="0"`, so users can create multiple resources with the same squid_opts.

Reply:
Thanks for your comment.
I fix the value of unique to "0"